iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/bucket/projectorg/read/DocumentOrganizationReader.java

```diff
@@ -0,0 +1,68 @@
+package eu.dnetlib.iis.wf.affmatching.bucket.projectorg.read;
+
+import java.io.Serializable;
+
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+
+import eu.dnetlib.iis.wf.affmatching.bucket.projectorg.model.AffMatchDocumentOrganization;
+import eu.dnetlib.iis.wf.affmatching.bucket.projectorg.model.AffMatchDocumentProject;
+import eu.dnetlib.iis.wf.affmatching.bucket.projectorg.model.AffMatchProjectOrganization;
+
+/**
+ * Reader of document-organization relations ({@link AffMatchDocumentOrganization}).<br />
+ * This reader internally uses {@link DocumentProjectReader} and {@link ProjectOrganizationReader}
+ * to read document-project and project-organization relations respectively and then
+ * combines them using {@link DocumentOrganizationCombiner}.
+ *  
+ * 
+ * @author madryk
+ */
+public class DocumentOrganizationReader implements Serializable {
```

Comment:
Hmm, this name suggests it reads relations from some source while in fact it combines them from different relations coming from two different sources.